apps/emqx_management/src/emqx_mgmt.erl

```diff
@@ -141,9 +141,28 @@ node_info() ->
         uptime => proplists:get_value(uptime, BrokerInfo),
         version => iolist_to_binary(proplists:get_value(version, BrokerInfo)),
         edition => emqx_release:edition_longstr(),
-        role => mria_rlog:role()
+        role => mria_rlog:role(),
+        log_path => log_path(),
+        sys_path => iolist_to_binary(code:root_dir())
     }.
 
+log_path() ->
+    RootDir = code:root_dir(),
+    Configs = logger:get_handler_config(),
+    case get_log_path(Configs) of
+        undefined ->
+            <<"log.file_handler.default.enable is false, not logging to file.">>;
+        Path ->
+            iolist_to_binary(filename:join(RootDir, Path))
+    end.
+
+get_log_path([#{config := #{file := Path}} | _LoggerConfigs]) ->
```

Comment:
This assumes we only have one logger handler that has file backend.

Reply:
it does not assume that, it returns the first one.